hrui/CRM/HRUI/Upgrader/Steps/4706.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+trait CRM_HRUI_Upgrader_Steps_4706 {
+
+  /**
+   * Adds "is applying for NI/SSN" field
+   */
+  public function upgrade_4706() {
+    $groupName = 'Inline_Custom_Data';
+    $customGroup = civicrm_api3('CustomGroup', 'get', ['name' => $groupName]);
+
+    if ($customGroup['count'] != 1) {
+      throw new \Exception("Cannot continue without Inline_Custom_Data group");
+    }
+
+    $customGroup = array_shift($customGroup['values']);
+
+    // Add "is applying" field
+    $fieldData = [
+      'custom_group_id' => $customGroup['id'],
+      'name' => 'has_applied_for_identification',
+      'label' => 'Applied for NI/SSN',
+      'html_type' => 'Radio',
+      'data_type' => 'Boolean',
+      'weight' => 2,
```

Comment:
The weight here, How do we know that 2 is the next weight to be added, or is there a function to get the max weight id and use it here or it's not necessary?